addons/website/views/snippets/s_map.xml

```diff
@@ -9,7 +9,7 @@
                     An address must be specified for a map to be embedded
                 </div>
             </div>
-            <iframe class="d-none s_map_embedded o_not_editable" width="100%" height="100%" src="about:blank" frameborder="0" scrolling="no" marginheight="0" marginwidth="0"></iframe>
+            <!-- GMap iframe will be injected in JS -->
```

Comment:
Not sure to understand? If the map snippet was dropped in a sanitized field, the page will be rendered and served without the iframe and injected in JS by the public widget after page load
If you it was not in a sanitized field, it will be injected in JS by the options during the edition and save in the page HTML. The page will then be served with the iframe.

In both cases, the JS is in charge of injecting the iframe at this place (let it be "persistent" or not).
The goal of the comment is for when someone reads the template to figure that at this place, there will be an iframe injected.

Not sure if it replies to your question :-)

Reply:
> Not sure if it replies to your question :-)

Not at all :smile: 

> The goal of the comment is for when someone reads the template to figure that at this place, there will be an iframe injected.

I understood that, hence I said: "I would not have added it but why not". Then I was wondering if the comment would be drag & drop in the page as well and saved with the page content and thus served to visitors as well. I checked, the comment is apparently part of the view:
![image](https://user-images.githubusercontent.com/10338094/169858703-5f3a74e8-2b7e-417b-a2a1-55a6aab7e5d9.png)
But stripped once rendered and thus not there in the DOM
![image](https://user-images.githubusercontent.com/10338094/169858869-1b6ce49d-4e1d-4014-8925-fbb0d1a938f7.png)
Good to know, I always avoided to add comments inside views thinking they would be rendered :grimacing:

Reply:
Oh, yeah I tripled checked that before adding it, didn't check what removed it or if it was kept in debug or something, but indeed it is not served to the client :+1: